### PR TITLE
update shard tensor for pp

### PR DESCRIPTION
**Pull request by zhaoyinglia**
### PR types

Others
### PR changes

Others
### Description

更新 shard 标注的位置，减少一次 send/recv

### CODE REVIEW

**Review by From00 — APPROVED** [review]
LGTM

File: model_zoo/gpt-3/ppfleetx/models/language_model/gpt/auto/auto_model.py [hunk 1]
```diff
@@ -291,6 +291,7 @@ def forward(self, tgt, memory, tgt_mask=None, memory_mask=None, use_cache=False,
 
         for i, mod in enumerate(self.layers):
             ipp = mod.ipp
+            auto.shard_tensor(output, auto_env.get_mesh()[ipp], [auto_env.get_mesh().dp_dim, None, None])
             mod = auto.shard_op(mod, auto_env.get_mesh()[ipp])
```
Comment (zhiqiu) [inline model_zoo/gpt-3/ppfleetx/models/language_model/gpt/auto/auto_model.py]: shard_op is not needed?

Comment (zhaoyinglia) [reply model_zoo/gpt-3/ppfleetx/models/language_model/gpt/auto/auto_model.py]: yes, deleted.

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**